projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-meta.php

```diff
@@ -306,7 +306,8 @@ public function test_get_object_by_id_size_limit_exceeded() {
 		update_post_meta( $this->post_id, $this->whitelisted_post_meta, $meta_test_value );
 
 		$module = Modules::get_module( 'meta' );
-		$metas  = $module->get_object_by_id( 'post', $this->post_id, $this->whitelisted_post_meta );
+		// @phan-suppress-next-line PhanParamTooMany -- Phan looks at the Module abstract class instead of the Meta class that extends it.
```

Comment:
Maybe better to tell Phan the right class?
```suggestion
		'@phan-var \Automattic\Jetpack\Sync\Modules\Meta $module';
```

Looks like the phpdoc on that `get_object_by_id()` method has the wrong typing for `$id` and `$meta_key` too, BTW.

Reply:
6906f662bc and ea69a4e11c